src/openzaak/components/catalogi/models/resultaattype.py

```diff
@@ -145,7 +145,9 @@ class ResultaatType(ETagMixin, OptionalGeldigheidMixin, models.Model):
         blank=True,
         help_text=_(
             "Naam van de attribuutsoort van het procesobject dat "
-            "bepalend is voor het einde van de procestermijn."
+            "bepalend is voor het einde van de procestermijn. "
+            "In het geval van afleidingswijze zaakobject kunnen nested attributen "
```

Comment:
This behavior is different from the standard. Could you please make a note about it here, perhaps mark the last sentence as experimental?

Reply:
added it to the helptext